src/components/CustomerDetails/CollectCustomerDetailsScreen.test.tsx

```diff
@@ -107,6 +109,8 @@ describe("CollectCustomerDetailsScreen", () => {
         </CreateProvidersWrapper>
       );
 
+      const continueAppButton = getByTestId(continueAppButtonId);
```

Comment:
since this screen is blocked, all tests require additional button click to reach original screen